test/e2e/volumes.go

```diff
@@ -370,7 +370,7 @@ var _ = framework.KubeDescribe("Volumes [Feature:Volumes]", func() {
 			config := VolumeTestConfig{
 				namespace:   namespace.Name,
 				prefix:      "nfs",
-				serverImage: "gcr.io/google_containers/volume-nfs:0.6",
+				serverImage: "gcr.io/google_containers/volume-nfs",
```

Comment:
I'd recommend not using `latest` tag. Instead specify an explicit tag which I assume will be `:0.7` in this case